cmd/nerdctl/version.go

```diff
@@ -102,13 +113,16 @@ func versionAction(cmd *cobra.Command, args []string) error {
 	return vErr
 }
 
-// versionInfo may return partial VersionInfo on error
-func versionInfo(cmd *cobra.Command, globalOptions types.GlobalCommandOptions) (dockercompat.VersionInfo, error) {
-
+// versionInfo may return partial VersionInfo on error.
+// Address can be empty to skip inspecting the server.
```

Comment:
Nice to have; Address can be a ptr type to avoid allocating usless memory space when empty

Reply:
The overhead is negligible